components/Plus/UserDetails.tsx

```diff
@@ -0,0 +1,87 @@
+import styled from "styled-components";
+import PlayArrowIcon from "@mui/icons-material/PlayArrow";
+import SellIcon from "@mui/icons-material/Sell";
+import CheckCircleIcon from "@mui/icons-material/CheckCircle";
+import HighlightOffIcon from "@mui/icons-material/HighlightOff";
+import PaymentIcon from "@mui/icons-material/Payment";
+import EventIcon from "@mui/icons-material/Event";
+import HourglassEmptyIcon from "@mui/icons-material/HourglassEmpty";
+
+export default function UserDetails({ user }) {
+  const Styled = {
+    DetailsContainer: styled.div`
+      display: flex;
+      justify-content: space-between;
+      padding: 8px 20px 0 20px;
+      border-bottom: 1px solid #ccc;
+    `,
+
+    DetailsData: styled.p`
+      flex: 1;
+    `,
+
+    IconTextWrapper: styled.div`
+      display: flex;
+      align-items: center;
+      gap: 8px;
+      margin-bottom: 16px;
+      &:not(:last-child) {
+        margin-bottom: 16px;
+      }
+    `,
+
+    HoverEffect: styled.div`
+      position: relative;
+      display: inline-block;
+
+      &:hover::after {
+        content: "${(props) => props.description}";
+        position: absolute;
+        bottom: -25px;
+        transform: translateX(-40%);
+        background-color: ${({ theme }) => theme.colors.grey2};
+        color: ${({ theme }) => theme.colors.white};
+        padding: 5px 10px;
+        border-radius: 5px;
+        font-size: 12px;
+        z-index: 1;
+      }
+    `,
+  };
+  return (
+    <Styled.DetailsContainer>
```

Comment:
I don't understand the concept with the Styled object and always using the `Styled.` notation (like mentioned in another comment). If this has any advantages please let me know.